src/config/topics.js

```diff
@@ -321,6 +327,26 @@ export const zweitausbildung = {
   searches: defaultSearches,
 };
 
+export const netzentwicklung = {
+  name: 'ch.sbb.netzentwicklung',
+  key: 'ch.sbb.netzentwicklung',
+  maxZoom: 13,
+  elements: { ...defaultElements, shareMenu: true, popup: true, overlay: true },
+  layers: [
+    kilometrageLayer,
+    netzentwicklungDataLayer,
+    netzentwicklungStrategischLayer,
+    netzentwicklungProgrammManagerLayer,
+    netzentwicklungSkPlanerLayer,
+  ],
+  projection: 'EPSG:3857',
+  // eslint-disable-next-line react/prop-types
+  layerInfoComponent: ({ language, t }) => (
```

Comment:
If we handle the language in i18n instead of a prop, we could make the topic more consistent with the other topics

Reply:
Done